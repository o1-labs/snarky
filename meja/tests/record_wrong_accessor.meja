type t('a, 'b, 'c) = {a : 'a, b : 'b, c : 'c};

let x = {a: 15, b: 20, c: 25};

module X = {
  type t('a) = {a : 'a, b : 'a, c : 'a};
};

let y = x.X.a;
