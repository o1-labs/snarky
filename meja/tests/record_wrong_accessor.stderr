File "tests/record_wrong_accessor.meja", line 3, characters 8-29:
Error: Incompatable types '_ X.t' and '(int, int, int) t':
Cannot unify ''a X.t' and '(int, int, int) t'.

